cmd/flowrunner/testdata/flows/brochure_test.json

```diff
@@ -165,9 +166,9 @@
                 {
                     "category": "",
                     "created_on": "2000-01-01T00:00:00.000000000-00:00",
+                    "input": "Ryan Lewis",
```

Comment:
we renamed operand to input to match RP on results, but we should also do here on the event